Description: Link to libgl2ps.so instead of compiling the internally
 shipped copy. Fix the 'embedded-library: gl2ps' lintian error.
Author: Daniel Leidert <dleidert@debian.org>
Forwarded: not-needed
Last-Update: 2013-05-13

--- a/libavogadro/src/extensions/CMakeLists.txt
+++ b/libavogadro/src/extensions/CMakeLists.txt
@@ -4,9 +4,7 @@
 
 set(DESTINATION_DIR ${Avogadro_PLUGIN_INSTALL_DIR}/extensions)
 
-include_directories(${CMAKE_CURRENT_BINARY_DIR}
-  "${CMAKE_CURRENT_SOURCE_DIR}/gl2ps"
-)
+include_directories(${CMAKE_CURRENT_BINARY_DIR})
 
 set(LINK_LIBS avogadro)
 set(PLUGIN_LABEL extensions)
@@ -113,8 +111,8 @@
 target_link_libraries(networkfetchextension ${QT_QTNETWORK_LIBRARY})
 
 # GL2PS
-avogadro_plugin(gl2psextension
-  "gl2psextension.cpp;gl2ps/gl2ps.c")
+avogadro_plugin(gl2psextension gl2psextension.cpp)
+target_link_libraries(gl2psextension gl2ps)
 
 # Subdirs
 add_subdirectory(crystallography)
